packages/theme-chalk/src/scrollbar.scss

```diff
@@ -15,14 +16,17 @@
   }
 
   @include e(wrap) {
-    overflow: scroll;
+    overflow: auto;
     height: 100%;
 
+    &::-webkit-scrollbar {
+      display: none;
```

Comment:
Like I said before, native scrollbar will display when `native=true`. Do not add this

Reply:
OK